validations/transactWriteItems.js

```diff
@@ -0,0 +1,194 @@
+var validations = require('./index'),
+  db = require('../db')
+
+exports.types = {
+  ReturnConsumedCapacity: {
+    type: 'String',
+    enum: [ 'INDEXES', 'TOTAL', 'NONE' ],
+  },
+  ReturnItemCollectionMetrics: {
+    type: 'String',
+    enum: [ 'SIZE', 'NONE' ],
+  },
+  ClientRequestToken: {
+    type: 'String'
+  },
+  TransactItems: {
+    type: 'List',
+    notNull: true,
+    lengthGreaterThanOrEqual: 1,
+    lengthLessThanOrEqual: 25,
+    children: {
+      type: 'ValueStruct<TransactWriteItem>',
+      children: {
+        Put: {
+          type: 'FieldStruct<Put>',
+          children: {
+            TableName: {
+              type: 'String',
+            },
+            ExpressionAttributeValues: {
+              type: 'Map<AttributeValue>',
+              children: 'AttrStruct<ValueStruct>',
+            },
+            ExpressionAttributeNames: {
+              type: 'Map<java.lang.String>',
```

Comment:
Is a Java reference an artefact of the validation library?
It's odd to see it here.